api/organisations/invites/models.py

```diff
@@ -60,14 +65,10 @@ class Meta:
         # reference existing table after moving to own app to avoid db inconsistencies
         db_table = "users_invite"
 
-    def save(self, *args, **kwargs):
-        # send email invite before saving invite
-        self.send_invite_mail()
-        super(Invite, self).save(*args, **kwargs)
-
     def get_invite_uri(self):
         return f"{get_current_site_url()}/invite/{str(self.hash)}"
 
+    @hook(AFTER_CREATE)
```

Comment:
Do we have a way for the user to send the invite email again (if the first one fails for some reason)?

Reply:
Gotcha